iquip/apps/builder.py

```diff
@@ -645,6 +722,23 @@ def argumentsFromListWidget(self, listWidget: QListWidget) -> Dict[str, Any]:
             args[widget.name] = widget.value()
         return args
 
+    def scannableFromListWidget(self, listWidget: QListWidget) -> Dict[str, Any]:
```

Comment:
Why did you define another method, not using `argumentsFromListWidget`?
They look identical.